faculty/clients/experiment.py

```diff
@@ -37,6 +37,12 @@ def __init__(self, message, conflicting_params=None):
             self.conflicting_params = conflicting_params
 
 
+class ExperimentNotActiveConflict(Exception):
```

Comment:
What about `ExperimentDeleted`?

The fact that it's a conflict doesn't seem relevant (that seems to mainly be a feature of the HTTP status code we're using) and 'Deleted' seems clearer (and closer to our data model) than 'NotActive'/